src/controllers/__test__/clientController.test.js

```diff
@@ -236,14 +250,68 @@ describe('getFavoriteDishes', () => {
 //   // testing getChefs
 // });
 
-// describe('getChefById',getChefs() => {
-//   // testing getChefById
-// });
+// describe('getChefById', () => {
+//   beforeEach(() => {
+//     jest.clearAllMocks();
+//   });
 
-// describe('dishesRatings',getChefs() => {
-//   // testing dishesRatings
+//   it('should return chef information by ID', async () => {
+//     const req = {
+//       params: { id: 'mockChefId' },
+//     };
+
+//     const mockChef = { name: 'Test Chef', specialty: 'Italian Cuisine' };
+
+//     Chef.findById.mockResolvedValue(mockChef);
+
+//     const res = {
+//       status: jest.fn().mockReturnThis(),
+//       json: jest.fn(),
+//     };
+
+//     await chefController.getChefById(req, res);
+
+//     expect(res.status).toHaveBeenCalledWith(200);
+//     expect(Chef.findById).toHaveBeenCalledWith(req.params.id);
+//     expect(res.json).toHaveBeenCalledWith(mockChef);
+//   });
 // });
```

Comment:
If you couldn't do this test, it's better to delete it rather than leave it as a comment.